MonoGame.Framework/Graphics/Texture3D.cs

```diff
@@ -74,11 +74,7 @@ protected Texture3D (GraphicsDevice graphicsDevice, int width, int height, int d
 		{
             ValidateParams(level, left, top, right, bottom, front, back, data, startIndex, elementCount);
 
-            var width = right - left;
```

Comment:
I figured i would fix bad platform apis when i could.

Here it was passing left, top, right, bottom, front, back....  AND width, height, depth. 

So i removed the extra data passing that not all platforms need.